app/services/start_price_thresholds.rb

```diff
@@ -0,0 +1,8 @@
+class StartPriceThresholds < Struct.new(:start_price)
+  MICROPURCHASE = 3500
+  SAT = 150000
+
+  def small_business?
```

Comment:
what if `SmallBusinessEligibility` had a ` small_business?` method that returned `true` and `InSamEligibility` returned `false` and we depended on those for this logic?